src/EpisodeManagementService/GetEpisode/GetEpisode.cs

```diff
@@ -0,0 +1,58 @@
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace NHS.ServiceInsights.EpisodeManagementService;
+public class GetEpisode
+{
+    private readonly ILogger<GetEpisode> _logger;
+    private readonly HttpClient _httpClient;
+
+    public GetEpisode(ILogger<GetEpisode> logger, HttpClient httpClient)
+    {
+        _logger = logger;
+        _httpClient = httpClient;
+    }
+
+    [Function("GetEpisode")]
+    public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get")] HttpRequestData req)
+    {
+        _logger.LogInformation("Request to retrieve episode information has been processed.");
+
+        string episodeId = req.Query["EpisodeId"];
+
+        if (string.IsNullOrEmpty(episodeId))
+        {
+            _logger.LogError("Please enter a valid Episode ID.");
+            return req.CreateResponse(HttpStatusCode.BadRequest);
+        }
+
+        // Call the Get Episode function in the Episode Data Service
+        var response = await _httpClient.GetAsync($"http://localhost:6070/api/GetEpisode?EpisodeId={episodeId}");
```

Comment:
URL shouldn't be hard coded I think, should be in local settings.

Reply:
We shouldn't be using HttpClient here, we should instead be using HttpRequestService which manages HttpClient for us

Reply:
Done